src/shared/adjusters/messages/en/increaseSize.json

```diff
@@ -6,5 +6,7 @@
     "additionalMagnifierAdjusters": "additional magnifier preferences",
     "additionalVisualAdjusters": "additional visual preferences",
     "moreMagnifierPreferences": "more magnifier preferences",
-    "lessMagnifierPreferences": "less magnifier preferences"
-}
\ No newline at end of file
+    "lessMagnifierPreferences": "less magnifier preferences",
```

Comment:
@dayotte  Do you mean the label should stay as it is?
See http://public.wsu.edu/~brians/errors/amount.html and http://www.quickanddirtytips.com/education/grammar/less-versus-fewer?page=all.